designcompose/src/main/java/com/android/designcompose/Shape.kt

```diff
@@ -0,0 +1,620 @@
+package com.android.designcompose
+
```

Comment:
And maybe call this file something that indicates it does path rendering, like ShapeRender or something?

Reply:
Yeah, good point. I think `ComputePaths` is a good name for the file because `computePaths` is the only publicly exported method. I should probably rename `ShapePaths` to `ComputedPaths` which sounds more like a derived value.

Reply:
Done. I think this makes it a lot clearer what it's doing and why you might need it.